android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -199,15 +208,6 @@ public void onUpdateExtractingVisibility(EditorInfo ei) {
     super.onUpdateExtractingVisibility(ei);
   }
 
-  @Override
-  public void onConfigurationChanged(Configuration newConfig) {
-    super.onConfigurationChanged(newConfig);
-    if (newConfig.orientation != lastOrientation) {
-      lastOrientation = newConfig.orientation;
-      KMManager.onConfigurationChanged(newConfig);
-    }
-  }
```

Comment:
It sounds as if this part of the code may be needed for getting proper landscape width, based upon https://github.com/keymanapp/keyman/pull/11604#issuecomment-2143665957.  But, with it here, it's possible for an update to `lastOrientation` here to block proper height-setting in `onStartInput`.